common/CommunityToolkit.Labs.Shared/Renderers/GeneratedSampleOptionsRenderer.xaml

```diff
@@ -16,28 +16,54 @@
         <local:GeneratedSampleOptionTemplateSelector x:Key="GeneratedSampleOptionTemplateSelector">
             <local:GeneratedSampleOptionTemplateSelector.BoolOptionTemplate>
                 <DataTemplate x:DataType="metadata:ToolkitSampleBoolOptionMetadataViewModel">
-                    <StackPanel>
-                        <TextBlock Text="{x:Bind Title}"
-                                   Visibility="{x:Bind local:GeneratedSampleOptionsRenderer.NullOrWhiteSpaceToVisibility(Title)}" />
-                        <CheckBox Content="{x:Bind Label}"
-                                  IsChecked="{x:Bind BoolValue, Mode=TwoWay}" />
-                    </StackPanel>
+                    <ToggleSwitch Header="{x:Bind Title, Mode=OneWay}"
+                                  IsOn="{x:Bind BoolValue, Mode=TwoWay}" />
                 </DataTemplate>
             </local:GeneratedSampleOptionTemplateSelector.BoolOptionTemplate>
             <local:GeneratedSampleOptionTemplateSelector.MultiChoiceOptionTemplate>
                 <DataTemplate x:DataType="metadata:ToolkitSampleMultiChoiceOptionMetadataViewModel">
-                    <StackPanel>
-                        <TextBlock Text="{x:Bind Title}"
-                                   Visibility="{x:Bind local:GeneratedSampleOptionsRenderer.NullOrWhiteSpaceToVisibility(Title)}" />
-                        <muxc:RadioButtons ItemsSource="{x:Bind Options}"
-                                           SelectedIndex="0"
-                                           SelectedItem="{x:Bind Value, Mode=TwoWay}" />
-                    </StackPanel>
+                    <ComboBox Header="{x:Bind Title, Mode=OneWay}"
```

Comment:
@niels9001 does this one work? I noticed the other `Title` are only `OneTime`, is that a difference?

Reply:
Unfortunately not :(